src/components/Toast/Toast.tsx

```diff
@@ -6,12 +6,18 @@ import {
   DEFAULT_TOAST_DURATION,
   FrameContext,
   frameContextTypes,
-  ToastProps as Props,
+  ToastProps,
 } from '../Frame';
 import {withAppProvider, WithAppProviderProps} from '../AppProvider';
 
 const createId = createUniqueIDFactory('Toast');
 
+// The script in the styleguide that generates the Props Explorer data expects
```

Comment:
My big motivation is make future-Ben think twice when he sees this in 6 months time, because I know he'll think "that looks pointless, I should tidy that up to be "ComposedProps = ToastProps & WithAppProviderProps" and then everything will break